### PR TITLE
task(branding): rename dashboard group to APM

### PR DESCRIPTION
**Pull request by pboster-splunk**
This changes the build-in dashbaord group from µAPM
to APM.

Issue: https://signalfuse.atlassian.net/browse/APMF-2155

### CODE REVIEW

File: group/APM 2.0.json [hunk 1]
```diff
@@ -1148,6 +1148,7 @@
         "options": {
           "colorBy": "Dimension",
           "colorScale2": null,
+          "hideMissingValues": false,
```
Comment (pboster-splunk) [inline group/APM 2.0.json:1151]: I didn't add this it came from the export from the UI

File: group/APM 2.0.json [hunk 2]
```diff
@@ -1245,6 +1246,7 @@
...
```
Comment (pboster-splunk) [inline group/APM 2.0.json:1249]: Same here UI export

File: group/APM 2.0.json [hunk 3]
```diff
@@ -3397,7 +3399,7 @@
             "dashboardGroupId": "EPK1YluAgAA",
             "dashboardId": "EPK1eRKAcAA",
             "isDefault": true,
-            "name": "µAPM 2.0 Service",
+            "name": "APM Service",
```
Comment (pboster-splunk) [inline group/APM 2.0.json:3402]: I don't know where the 2.0 came from - do I need to add the 2.0?

File: group/APM 2.0.json [hunk 4]
```diff
@@ -3753,17 +3755,17 @@
         "created": 0,
         "creator": null,
         "customProperties": null,
-        "description": "Service-level indicators from µAPM Tracing Metrics",
+        "description": "Service-level indicators from APM Tracing Metrics",
         "discoveryOptions": null,
-        "eventOverlays": null,
+        "eventOverlays": [],
```
Comment (pboster-splunk) [inline group/APM 2.0.json:3760]: UI export again

File: group/Smart Gateway.json [hunk 5]
```diff
@@ -1,3018 +1,3291 @@
 {
-  "chartExports" : [ {
```
Comment (pboster-splunk) [inline group/Smart Gateway.json:2]: This diff makes it look like I changes everything but the JSON formatting changed and numbers with decimals got round to whole number like 10.0 -> 10.  I did not do that.  I just changed µAPM to APM in the context for the first chart.  I turned off whitespace and new lines and the diff was readable on my IDE.

File: group/Smart Gateway.json [hunk 6]
```diff
@@ -1,3018 +1,3291 @@
...
-      "creator" : null,
-      "customProperties" : { },
-      "description" : "",
-      "id" : "DzaUUdfAgAA",
-      "importOf" : "Dy6Kf3KAEAE",
-      "lastUpdated" : 0,
-      "lastUpdatedBy" : null,
-      "name" : ".",
-      "options" : {
-        "markdown" : "<table width=\"100%\" height=\"30%\" rules=\"none\"><tr><td valign=\"middle\" align=\"center\" bgcolor=\"#4682B4\">\n<font size=\"5\" color=\"white\">Internal Smart Gateway Metrics</font>\n</td></tr></table><br>\nThis dashboard requires turning on the sending of [Internal Gateway Metrics] (https://github.com/signalfx/integrations/tree/master/gateway#signalfx-performance-options) by setting the `EmitDebugMetrics` parameter in your Smart Gateway configuration.\n\n_Note that these metrics will take up some of your Custom Metrics and DPM capacity and are **not** included in your µAPM plan._",
```
Comment (pboster-splunk) [inline group/Smart Gateway.json:1153]: This what I changed in the UI.  I'll mark the update too.

File: group/Smart Gateway.json [hunk 7]
```diff
@@ -1,3018 +1,3291 @@
...
+        "creator": null,
+        "customProperties": {},
+        "description": "",
+        "id": "DzaUUdfAgAA",
+        "importOf": "Dy6Kf3KAEAE",
+        "lastUpdated": 0,
+        "lastUpdatedBy": null,
+        "name": ".",
+        "options": {
+          "markdown": "<table width=\"100%\" height=\"30%\" rules=\"none\"><tr><td valign=\"middle\" align=\"center\" bgcolor=\"#4682B4\">\n<font size=\"5\" color=\"white\">Internal Smart Gateway Metrics</font>\n</td></tr></table><br>\nThis dashboard requires turning on the sending of [Internal Gateway Metrics] (https://github.com/signalfx/integrations/tree/master/gateway#signalfx-performance-options) by setting the `EmitDebugMetrics` parameter in your Smart Gateway configuration.\n\n_Note that these metrics will take up some of your Custom Metrics and DPM capacity and are **not** included in your APM plan._",
```
Comment (pboster-splunk) [inline group/Smart Gateway.json:1240]: Here's where I changed µAPM to APM

**Comment by mpetazzoni** [comment]
@pboster-splunk Looks good to me, but you need to change the base branch to be `lab` so this PR gets merged into the `lab` branch and not into `master`.

**Comment by pboster-splunk** [comment]
@mpetazzoni I changed to be based on `lab` branch

**Comment by pboster-splunk** [comment]
@mpetazzoni Did you want to review the diffs again?  They look the same as when it was against `master` branch.

**Review by mpetazzoni — APPROVED** [review]

File: group/APM 2.0.json (hunk 3 above)
Comment (pboster-splunk) [reply group/APM 2.0.json:3402]: Apparently this is OK

**Pull request merged**